python/requirements.txt

```diff
@@ -4,6 +4,7 @@ bottle==0.12.25
 cheroot==10.0.0
 gpsdclient==1.3.2
 grpcio==1.60.0
+json5
```

Comment:
Not that requirements are added to frequently, but for potential future reference, I prefer to have specific versions listed here. I'll update this after merging :+1: